_maps/map_files/cyberiad/z3.dmm

```diff
@@ -1,357 +1,234 @@
-"dv" = (/obj/structure/table,/obj/item/storage/fancy/cigarettes,/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dw" = (/obj/structure/stool/bed/chair{dir = 8},/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dx" = (/obj/structure/window/reinforced{dir = 1},/obj/structure/window/reinforced{dir = 8},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 6},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "vault"; dir = 5},/area/turret_protected/tcomsat)
-"dy" = (/obj/structure/window/reinforced,/obj/structure/window/reinforced{dir = 1},/obj/machinery/atmospherics/pipe/manifold/hidden/supply{level = 1},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "vault"; dir = 5},/area/turret_protected/tcomsat)
-"dz" = (/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4},/obj/structure/window/reinforced,/obj/structure/cable{d1 = 1; d2 = 4; icon_state = "1-4"; tag = ""},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "vault"; dir = 5},/area/turret_protected/tcomsat)
-"dA" = (/obj/structure/window/reinforced,/obj/structure/window/reinforced{dir = 1},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0; tag = ""},/turf/simulated/floor/plasteel{tag = "icon-vault (NORTHEAST)"; icon_state = "vault"; dir = 5},/area/turret_protected/tcomsat)
-"dB" = (/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 9; level = 1},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0; tag = ""},/obj/machinery/door/airlock/hatch{name = "Telecoms Control Room"; req_access_txt = "61"},/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dC" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0; tag = ""},/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dD" = (/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"; tag = ""},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dE" = (/obj/machinery/power/apc{dir = 4; name = "Telecoms Sat. Control APC"; pixel_x = 25},/obj/structure/cable{d2 = 8; icon_state = "0-8"},/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dF" = (/obj/machinery/light{dir = 8},/obj/structure/table,/obj/item/multitool,/obj/structure/sign/electricshock{pixel_x = -32},/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dG" = (/obj/effect/spawner/window/reinforced,/obj/structure/cable{icon_state = "0-4"; d2 = 4},/turf/simulated/floor/plating,/area/tcommsat/chamber)
-"dH" = (/turf/simulated/wall/r_wall,/area/tcommsat/chamber)
-"dI" = (/obj/machinery/vending/snack,/turf/simulated/floor/plasteel,/area/tcommsat/computer)
-"dJ" = (/obj/machinery/vending/cola,/obj/machinery/atmospherics/pipe/simple/hidden/supply,/turf/simulated/floor/plasteel,/area/tcommsat/computer)
```

Comment:
Just because you never seen this room used, it does not mean it can be thrown out. Please place something like that back.

Reply:
The break room literally has no use. Give me a good reason to have a break room there when no one ever uses thes at

Reply:
What do you mean no-one ever uses the sat? I've been there a million times and believe it or not, the room sees use.

Reply:
Around a year ago, for 2 months straight I went to the sat every round on some snowflake job. I never used that room, aside from that one time that I am not going to talk about